tests/unittests/OnnxImporterTest.cpp

```diff
@@ -5437,3 +5437,127 @@ TEST_F(OnnxImporterTest, softmax13) {
+  ExecutionEngine EE;
+  auto &mod = EE.getModule();
+  std::string netFilename(GLOW_DATA_PATH
+                          "tests/models/onnxModels/mixedPrecision.onnxtxt");
+  PlaceholderBindings bindings;
+  auto *F = mod.createFunction("main");
+  Placeholder *output;
+  Tensor data(ElemKind::FloatTy, {1, 3, 28, 28});
+
+  // Precision config info used in model loader
+  std::string precisionConfigFile(
+      GLOW_DATA_PATH "tests/models/onnxModels/mixedPrecision.yaml");
+  setModelLoaderPrecisionOpt(precisionConfigFile);
+
+  {
+    ONNXModelLoader onnxLD(netFilename, {"input"}, {&data.getType()}, *F);
+    output = EXIT_ON_ERR(onnxLD.getSingleOutput());
+    EXPECT_EQ(mod.getPlaceholders().size(), 2);
+    bindings.allocate(mod.getPlaceholders());
+  }
+  EE.compile(CompilationMode::Infer);
+  EE.run(bindings);
+  setConstantFoldLoaderOpsFlag(false);
+  auto result = bindings.get(output)->getHandle();
+  std::vector<dim_t> expectedDims = {1, 16, 26, 26};
+  EXPECT_TRUE(result.dims().vec() == expectedDims);
+
+  // This Model has three operators
+  // Here Conv and ReLu are specified in the yaml file to run in fp16.
+  //  Input: Placeholder(float)
+  //      |
+  //      V
+  //  Conv (set to float16)
+  //      |
+  //      V
+  //  BatchNorm(float)
+  //      |
+  //      V
+  //  ReLu( set to float16)
+  //      |
+  //      V
+  //  Output: Placeholder(float)
+  // Check in the Glow graph the corresponding nodes are in FP16.
+  for (auto &n : F->getNodes()) {
+    if (n.getName().str() == "Relu_2__1_max__1" ||
+        n.getName().str() == "Conv_0__2") {
+      EXPECT_TRUE(n.getElementType(0) == ElemKind::Float16Ty);
+    }
+  }
+}
+
+TEST_F(OnnxImporterTest, mixedPrecisionConfigMultipleUsers) {
+  // This test checks when a list of node instances set to run in fp16
+  // has multiple users among which one user is not specified to run in fp16
+  ExecutionEngine EE;
+  auto &mod = EE.getModule();
+  std::string netFilename(
+      GLOW_DATA_PATH "tests/models/onnxModels/mixedPrecisionMultUsers.onnxtxt");
+  PlaceholderBindings bindings;
+  auto *F = mod.createFunction("main");
+  Placeholder *output;
+  Tensor data(ElemKind::FloatTy, {1, 3, 28, 28});
+
+  // Precision config info used in model loader
+  std::string precisionConfigFile(
+      GLOW_DATA_PATH "tests/models/onnxModels/mixedPrecisionMultUsers.yaml");
+  setModelLoaderPrecisionOpt(precisionConfigFile);
+
+  {
+    ONNXModelLoader onnxLD(netFilename, {"input"}, {&data.getType()}, *F);
+    output = EXIT_ON_ERR(onnxLD.getSingleOutput());
+    EXPECT_EQ(mod.getPlaceholders().size(), 2);
+    bindings.allocate(mod.getPlaceholders());
+  }
+  EE.compile(CompilationMode::Infer);
+  EE.run(bindings);
+  setConstantFoldLoaderOpsFlag(false);
+
+  auto result = bindings.get(output)->getHandle();
+  std::vector<dim_t> expectedDims = {1, 16, 26, 26};
+  EXPECT_TRUE(result.dims().vec() == expectedDims);
+
+  // This Model has five operators
```

Comment:
We should have ConvertTo nodes in the graph too, right? Perhaps include in the picture/verify they're there as expected?

Reply:
Done

Reply:
nit: I think this isn't actually done right? I was thinking we'd have `ConvertTo` as part of the visualization below.

Reply:
Previous visualization was for the original graph, now changed to the glow graph.